plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -278,13 +278,14 @@ public void executeUpdate(QueryJobConfiguration job)
         }
     }
 
-    public TableResult query(String sql, boolean useQueryResultsCache, CreateDisposition createDisposition)
+    public TableResult query(String sql, boolean useQueryResultsCache, boolean allowLargeResults, CreateDisposition createDisposition)
     {
         log.debug("Execute query: %s", sql);
         try {
             return bigQuery.query(QueryJobConfiguration.newBuilder(sql)
                     .setUseQueryCache(useQueryResultsCache)
                     .setCreateDisposition(createDisposition)
+                    .setAllowLargeResults(allowLargeResults)
```

Comment:
The javadocs mentions "If true setDestinationTable(TableId) must be provided", but this method doesn't call the setter. Is this change really effective?

Reply:
I was going through the example: https://cloud.google.com/bigquery/docs/samples/bigquery-query-legacy-large-results

Reply:
Reminder.